### ISSUE TITLE
Add '--custom-search' for help_crack.py

### ISSUE THREAD

**Issue by lugery** [RealEnder/dwpa#71]
Hello, 
There is something I want so much🙈

'--custom' ->'custom user dictionary to be checked against all uncracked handshakes'
Dreaming of the search bar on the site😇
'--custom-search' ->'and searched handshakes(Bssid/Ssid)' 

(Custom user dict and all uncracked handshakes and searched handshakes(Sql code order by asc) , Great group of 3)

Sample: --custom word.txt --custom-search inwi%            
(inwi% -> ssid)

Sample2 : --custom word.txt --custom-search e0:22:04:8f:3c:21
(e0:22:04:8f:3c:21 -> bssid) 

This search function exists on the site and I hope it can be easily added here😇
Such a feature excites me😊

Note: My English is not good. But I tried to explain 😇🙈

**Comment by ZerBea** [comment]
That would be a nice improvement if the user additional can select the networks either by ESSID, part of ESSID or MAC_AP and run a custom wordlist task against them.
It will make wpa-sec (wordlists) a little bit more dynamic.

**Comment by FFAMax** [comment]
> can select the networks either by ESSID, part of ESSID or MAC_AP and run a custom wordlist task against them

Yep, have this feature to work with custom dict should be great along with rules https://github.com/RealEnder/dwpa/issues/112
